### PR TITLE
Added RULES!!!

### PR DESCRIPTION
**Pull request by kingoftheconnors**
The Codex section for RULES now includes:
- rules_income (overrides existing so rules don't say that villages give gold)
- rules_capture (overrides existing so rules don't say that destroyed structures can be recaptured)
- rules_strongholds (overrides existing so rules to add that structures can recruit Villagers)
- rules_gold_mines
- rules_tech
- rules_building
- rules_repairing
- rules_pop_cap
- rules_artifacts
- rules_shop

### CODE REVIEW

File: assets_src/config/gameplay/codex/codex_rules.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  icon:
+    image: ui/codex/icons/icon_rules_question.png
+  pages:
+  - pageType: page
+    layoutType: animation
+    animation: ui/codex/content/rules_shop
+    title: codex_rules_shop_title
+    text: codex_rules_shop_text
+
+# We also need a REPAIRING rule
```
Comment (kingoftheconnors) [inline assets_src/config/gameplay/codex/codex_rules.yaml]:
```suggestion
```
Hey, past self! You've already made the Repairing codex entry!

Comment (kingoftheconnors) [reply assets_src/config/gameplay/codex/codex_rules.yaml]: Oh. Thanks, future self!

File: assets_src/config/strings/en-GB.yaml [hunk 2]
```diff
@@ -50,4 +50,39 @@ en-GB:
     unit_description_groove_pot: Filled with weird blue goo, increases groove by 25 (points, not percentage)
     unit_description_bow: A bow used by skilled hunter. Allows the wielder to use ranged attack at half the damage
     unit_description_dimensional_door: A relic from the anicent. Allows the wielder to teleport to a tile beside HQ. 5 turn cooldown
+
+    codex_rules_gold_mines: Gold Mines
+    codex_rules_gold_mines_title: Gold Mines
+    codex_rules_gold_mines_text: "> Mines take x1.3 damage\n> Mines have a finite amount of gold that can be mined from them.\n> When a gold mine is almost empty, a yellow warning will appear over the mine.\n> When a gold mine is completely empty, a red exclamation mark will appear, signaling that gold can no longer be gained from that gold mine.\n> Gold mines have a max of 4000 gold.\n> Gem mines have a max of 3000 gold.\n> You can see what percentage of gold is left based on the gold pile's health percentage"
+
+    codex_rules_tech: Tech
+    codex_rules_tech_title: Increasing Tech Level
```
Comment (Pokachi) [inline assets_src/config/strings/en-GB.yaml:59]: wight want to call out the gold required for teching up

File: assets_src/config/strings/en-GB.yaml [hunk 3]
```diff
@@ -50,4 +50,39 @@ en-GB:
...
+
+    codex_rules_shop: Shops
+    codex_rules_shop_title: Buying Artifacts
+    codex_rules_shop_text: "> Artifacts can be bought by Commanders (and only Commanders) at shops.\n> Shops sell five random artifacts and restock every 5 turns"
+
+    codex_rules_repair: Repairing
+    codex_rules_repair_title: Repairing Structures
+    codex_rules_repair_text: "> Villagers can repair structures to improve their sustainability.\n> Repairing costs 10 gold and restores up to 20% health.\n> Because structures no longer regenerate health each turn, repairing is the only way for structures to regain health."
```
Comment (Pokachi) [inline assets_src/config/strings/en-GB.yaml]: repairing costs half of building's original cost * the amount of health repaired

File: assets_src/config/strings/en-GB.yaml [hunk 4]
```diff
@@ -50,4 +50,39 @@ en-GB:
...
+
+    #These override existing rules pages to reflect changes in structures
+    codex_rules_income_text: "> Gold Mines and Gem Mines provide an income based on the number of villagers in each mine.\n> Gold mines are worth 80 gold per villager per turn.\n> Gem Mines are worth 120 gold per villager per turn.\n> Each mine can only hold 4 villagers at a time.\n> If a mine is destroyed, all mining villagers die.\n> Starting funds are set on a map-by-map basis."
```
Comment (Pokachi) [inline assets_src/config/strings/en-GB.yaml]: 3 villagers

**Review by Pokachi — APPROVED** [review]

**Pull request merged**